pkg/virt-controller/watch/application.go

```diff
@@ -658,6 +661,9 @@ func (vca *VirtControllerApp) initCommon() {
 		topologyHinter,
 		netAnnotationsGenerator,
 		netvmicontroller.UpdateStatus,
+		func(field *k8sfield.Path, vmiSpec *v1.VirtualMachineInstanceSpec, clusterCfg *virtconfig.ClusterConfig) []metav1.StatusCause {
+			return netadmitter.ValidateCreation(field, vmiSpec, clusterCfg)
+		},
```

Comment:
Could this function be assigned to a variable above, like `topologyHinter`?
Just to make the code more readable and give the function a proper name.

Reply:
I guess it is possible, I have no strong opinion on this.
Generally speaking, the wrapping anonymous function acts as a translation layer for the concrete and interface arguments, and has no real meaning beyond it. It has no abstraction naming over the name you see now, which is to validateCreation for network.

Reply:
@jean-edouard as Edy mentioned exporting it variable wont improve naming because its thin wrapper for injecting the concrete function,  WDYT?

Reply:
No strong opinion here either
/unhold